project-api-tests/src/test/java/org/ballerina/projectapi/CentralTestUtils.java

```diff
@@ -163,10 +181,22 @@ static String getGenerateBalaLog(String org, String pkgName, String platform, St
         return "Creating bala\n" + "\ttarget/bala/" + org + "-" + pkgName + "-" + platform + "-" + version + ".bala";
     }
 
+    /**
+     * Get the log output when package is successfully pushed to Central.
+     *
+     * @param orgName Organization name
+     * @param pkgName
```

Comment:
Shall we add a description for `pkgName`?
Is it okay to keep `return` empty in L190?

Reply:
Updated with e697e99dbd864835f445fa6942c2f2b0792c64b5